### PR TITLE
docs: add documentation for log highlighting

### PR DESCRIPTION
**Pull request by rangoo94**
## Pull request description 

* add documentation for log highlighting

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: docs/docs/testkube-pro/articles/log-highlighting.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## Overview
+
+In Testkube Pro, we highlight relevant keywords in logs for faster debugging. To use this feature, open execution details.
+
+On this screen, all of the lines that may be relevant will be highlighted in the interface.
+
+![log-highlighting.png](../../img/log-highlighting.png)
+
+You may navigate through the highlighted lines with the arrows on top of the interface,
```
Comment (jfermi) [inline docs/docs/testkube-pro/articles/log-highlighting.md:21]:
```suggestion
You may navigate through the highlighted lines with the arrows on top of the interface
```

File: docs/docs/testkube-pro/articles/log-highlighting.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+or using the scrollbar where all relevant lines are marked.
```
Comment (jfermi) [inline docs/docs/testkube-pro/articles/log-highlighting.md:22]:
```suggestion
or use the scrollbar where all relevant lines are marked.
```

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

**Comment by rangoo94** [comment]
Thank you @jfermi, could you also take a look at the newly added "Filtering" section? Thanks in advanced!

File: docs/docs/testkube-pro/articles/log-highlighting.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+On this screen, all the lines that may be relevant will be highlighted in the interface.
+
+![log-highlighting.png](../../img/log-highlighting.png)
+
+You may navigate through the highlighted lines with the arrows on top of the interface
+or use the scrollbar where all relevant lines are marked.
+
+## Filtering
+
+To decide on the active highlight categories, you may click "Highlight for keywords" button.
```
Comment (jfermi) [inline docs/docs/testkube-pro/articles/log-highlighting.md:26]:
```suggestion
To decide on the active highlight categories, you may click the "Highlight for keywords" button.
```

**Review by jfermi — COMMENTED** [review]
An editing suggestion.

**Review by haneabogdan — APPROVED** [review]

**Pull request merged**